packages/contracts/src/plugins/governance/majority-voting/token/build-metadata.json

```diff
@@ -1,13 +1,105 @@
-      "tuple(address addr, string name, string symbol) tokenSettings",
-      "tuple(address[] receivers, uint256[] amounts) mintSettings"
-    ],
-    "prepareUpdate": [""],
-    "prepareUninstallation": [""]
+    "prepareInstallation": {
+      "description": "The information required for the installation of build 2.",
+      "inputs": [
+        {
+          "components": [
+            {
+              "internalType": "enum MajorityVotingBase.VotingMode",
+              "name": "votingMode",
+              "type": "uint8",
+              "description": "A parameter to select the vote mode. In standard mode (0), early execution and vote replacement are disabled. In early execution mode (1), a proposal can be executed early before the end date if the vote outcome cannot mathematically change by more voters voting. In vote replacement mode (2), voters can change their vote multiple times and only the latest vote option is tallied."
+            },
+            {
+              "internalType": "uint32",
+              "name": "supportThreshold",
+              "type": "uint32",
+              "description": "The support threshold value. Its value has to be in the interval [0, 10^6] defined by `RATIO_BASE = 10**6`."
+            },
+            {
+              "internalType": "uint32",
+              "name": "minParticipation",
+              "type": "uint32",
+              "description": "The minimum participation value. Its value has to be in the interval [0, 10^6] defined by `RATIO_BASE = 10**6`."
+            },
+            {
+              "internalType": "uint64",
+              "name": "minDuration",
+              "type": "uint64",
+              "description": "The minimum duration of the proposal vote in seconds."
+            },
+            {
+              "internalType": "uint256",
+              "name": "minProposerVotingPower",
+              "type": "uint256",
+              "description": "The minimum voting power required to create a proposal."
+            }
+          ],
+          "internalType": "struct MajorityVotingBase.VotingSettings",
+          "name": "_votingSettings",
+          "type": "tuple",
+          "description": "The majority voting settings that will be applied as parameters on proposal creation."
+        },
+        {
+          "components": [
+            {
+              "internalType": "address",
+              "name": "token",
+              "type": "address",
+              "description": "The token address. If this is `address(0)`, a new `GovernanceERC20` token is deployed. If not, the existing token is wrapped as an `GovernanceWrappedERC20`."
+            },
+            {
+              "internalType": "string",
+              "name": "name",
+              "type": "string",
+              "description": "The token name. This parameter is only relevant if the token address is `address(0)`."
```

Comment:
"When you choose to create a new token, use this field to choose a name for it"

Or similar